guides/v2.1/frontend-dev-guide/translations/translate_practice.md

```diff
@@ -1,66 +1,63 @@
 ---
 group: frontend-developer-guide
-title: Create a translation dictionary for a theme
+title: Example theme translation dictionary
```

Comment:
Make sure you search for instances of the old title across all of devdocs, since the new title isn't similar to the old.